src/Console/src/Application/Executor.php

```diff
@@ -94,18 +94,18 @@ public function run(Section $section, $argv = []): void
 
         if ($this->matchedSection === null || $this->matchedInput === null) {
             $this->invalidUsage($argv);
-            return;
+            return 1;
         }
 
         if ($this->beforeAction()->isActionPrevented()) {
-            return;
+            return 0;
         }
 
         if (($action = $this->matchedSection->getAction()) === null) {
-            return;
+            return 0;
```

Comment:
what do other clis do when executing a section with ought the command? I'd have thought it was a 1 to signify it's not a command?

Reply:
git and composer return 1. not sure when this condition gets invoked though `ws asd asd asd` returns 1 (invalidUsage)

Reply:
but yeah, the above 2 should probably be non-zero